response_to_reviewers.Rmd

```diff
@@ -0,0 +1,32 @@
+
+Although we acknowledge that some variable names are not entirely descriptive, we've kept many of the names in order to be conscise in our coding style. However, where necessary, we have included more descriptive comments to better help guide our readers. Along those same lines, in our comments we've attempted to better describe the places in our code where we are merely checking our assumptions ahead of statistical analyses and where we are performing tests to directly compare our output to that from the original article. Also, some of the variable naming, particularly within functions, e.g.: "pwr.chisq.test()", was specific to the command structure of our analyses; and so we could not change them. However, we do strongly encourage the use of the R-helper shortcuts (i.e., "??[function here]") for further review of the command structures for functions, should our readers have questions about what certain abbreviations stand to represent. 
+
+As one reviewer pointed out, some code for the logistic regression was superfulous/redundant to analyses. We have deleted the redundancies while also preserving the integrity of the analyses. 
+
+Also, we were asked by a reviewer to consider plotting the results of our logistic regression by gender at the bottom of our report. While we enthusiastically believe in the importance of visualizations, we also recognize the recommendations of many other researchers regarding the consequence of plotting a categorical predictor (condition) against a binary outcome (pen choice) without any mixed-effects: compared to graphs with continuous predictors, a graph of our analyses would contribute much less to readers' understanding of the null finding. As a result, we've amended the parameters of our RMD file in order to ensure that the predicted probabilties for each gender print out clearly for the reader to see and compare directly. Furthermore, we have included summarizing paragraphs for all our statistical analyses. These paragraphs follow the code chunks representing our efforts for each analysis.
+
+And finally, upon the recommendation of several reviewers, we've added an image of the original figure, a table of frequencies, that we sought to reproduce in this project. Please note that this table is an abbreviated version, amended only slightly from the published version to relfect only the information pertinent to the study we replicate (the original contains results from studies for which we lacked data).
+
+Thanks again to all the eyes we've had on our report. We sincerely believe that all the feedback, encouragement and recommendations have resulted in the strongest version of our report possible. And for that, we are grateful!
+
+Regards, 
+
+Yaritza Perez-Hooks & Kyonne Isaac
```

Comment:
Wonderful letter.  This is very good style for handling feedback, even when you might not agree with all of it.  :+1: